src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -227,7 +227,7 @@ void InitializeShortcutBar()
         {
             ShortcutToolbar shortcutBar = new ShortcutToolbar(this.dynamoViewModel.Model.UpdateManager);
             shortcutBar.Name = "ShortcutToolbar";
-
+            Library.Text = Properties.Resources.LibraryText;
```

Comment:
A Window is as the name suggests a window, it can be closed, minimized, resized etc. This should be quite intuitive.

A UserControl on the other hand is a composite component/module which can be placed inside other controls and is itself made up of controls (possibly even other UserControls), the main use for UserControls is reusability, encapsulation and loose coupling, some applications can be broken up into a set of UserControls of which each one provides a certain functionality.